client/packages/common/src/intl/locales/en/common.json

```diff
@@ -132,6 +132,7 @@
   "label.auth-status": "Approval Status",
   "label.available-packs": "Available (packs)",
   "label.available-quantity": "Available quantity: {{number}} units",
+  "label.available-quantity-for-return": "Quantity Available for Return",
```

Comment:
Wonder if these should be in `distribution.json`?

Reply:
These are column labels, which are provided from the common package, so they do need to be here, but I agree its a bit confusing... I wonder if we could distinguish them with "common.{bla}" rather than just "label."?